static/docs/user-guide/update-tracked-file.md

```diff
@@ -1,10 +1,14 @@
 # Update a Tracked File
 
 Due to the way DVC handles linking between the data files in the cache and their
-counterparts in the working directory (see
-[#799](https://github.com/iterative/dvc/issues/799) and
-[#599](https://github.com/iterative/dvc/issues/599) for example), updating
-tracked files has to be carried out with caution.
+counterparts in the working directory (refer to [Cache File
+Linking](/docs/user-guide/cache-file-linking)), updating tracked files has to be
+carried out with caution to avoid data corruption when the DVC config option
+`cache.type` is set to `hardlink` or/and `symlink`. (See `dvc config cache` for
+more details on setting the cache file link types.)
```

Comment:
is it `).`?

Reply:
In this case the entire sentence is inside the parenthesis so `.)` is correct 😃

I double checked just now and found several sources verifying this e.g. https://www.quickanddirtytips.com/education/grammar/periods-and-parentheses

```
If the words inside the parentheses are a complete sentence, the period, question
mark, or exclamation point that ends the sentence goes inside the parenthesis:

Bring chocolate. (Squiggly likes sweets.)
```